x/emissions/proto/emissions/v3/genesis.proto

```diff
@@ -219,6 +219,12 @@ message TopicIdActorIdScore {
   Score Score = 3;
 }
 
+message TopicIdActorIdUint64 {
```

Comment:
I suspect this should not be here and should be only in v5/genesis.proto

it does not make sense why we are editing old protos

someone please justify this to me if it's necessary else lets make sure this is in v5, and if so delete from here

Reply:
removed